plugins/source/aws/resources/services/fsx/data_repository_tasks.go

```diff
@@ -71,6 +71,16 @@ func DataRepositoryTasks() *schema.Table {
 				Type:     schema.TypeJSON,
 				Resolver: schema.PathResolver("FailureDetails"),
 			},
+			{
+				Name:     "file_cache_id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("FileCacheId"),
+			},
+			{
+				Name:     "file_system_id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("FileSystemId"),
+			},
```

Comment:
Right now, with PostgreSQL being the only destination, I don't think it breaks backwards compatibility... However as we add more destinations this may become an issue (e.g. for a CSV destination it absolutely will break, and possibly for some other data lakes too)